content/docs/reference-events.md

```diff
@@ -165,6 +165,14 @@ onFocus onBlur
 
 These focus events work on all elements in the React DOM, not just form elements.
 
+#### onFocus
+
+The `onFocus` prop is initiated when focus is on the element. For example, when the user clicks on a text input element, the function defined with the `onFocus` prop is called.
```

Comment:
That is certainly true for chrome on windows and linux as far as I know. But does not apply to e.g. firefox macOS. 

More specifically: By spec it is up to the user agent (e.g. a browser) to determine whether a clicked element is also focused.

Reply:
Or it might be true for inputs. I was a bit suspicious because of the "click-focus" association which is a bit tricky on other platforms.

Reply:
@eps1lon I just booted up Firefox + Mac and ran through a sample:

https://www.loom.com/share/34f1a18300b44187b7a65e2c01b73f8b

Sandbox: https://codesandbox.io/s/busy-browser-0me16?file=/src/App.js:146-191

It _appears_ to be 1:1 to Chrome, at least for the basic examples I wanted to document. If you find something else different please feel free to edit/tack on.